LongWarOfTheChosen/Src/LW_Overhaul/Classes/LW2_UIImageSelector.uc

```diff
@@ -0,0 +1,294 @@
+// this is mostly the same as UIColorSelector
+class LW2_UIImageSelector extends UIPanel;
+
+var float ITEM_PADDING;
+var float EDGE_PADDING;
+var float				ChipHeight; 
+var float				ScrollbarPadding;
+var UIScrollbar		Scrollbar;
+var UIMask				Mask;
+var UIPanel			ChipContainer;
+var array<LW2_UISquadImage> ImageChips; 
+var array<string>		Images;
+var int				InitialSelection; 
```

Comment:
Uneven whitespace

Reply:
Could just be uneven in GitHub. I'll check. (GitHub displays tabs as 8 characters, whereas the text editor should be set up for 4 characters).